internal/cmd/controller/gitops/reconciler/gitjob_controller.go

```diff
@@ -57,6 +57,8 @@ const (
 
 	defaultPollingSyncInterval = 15 * time.Second
 	gitPollingCondition        = "GitPolling"
+	generationLabel            = "generation"
+	forceSyncGenerationLabel   = "forceSyncGeneration"
```

Comment:
```suggestion
	forceSyncGenerationLabel   = "fleet.cattle.io/force-sync-generation"
```

All our labels seem to use kebap-case